tests/test_utils.py

```diff
@@ -274,20 +281,24 @@ def test_standard_zip_dir_with_env_variable(
             os.path.join("subdir", os.path.basename(file_in_a_subdir))
             not in zip_file.namelist()
         )
-        assert os.path.join(self.FILE_IN_SECOND_TEST_DIRECTORY) in zip_file.namelist()
+        assert Path(file_in_other_directory).name in zip_file.namelist()
 
-    def test_get_snowflake_packages(self, streamlit_requirements_txt):
-        os.chdir(".tests")
+    def test_get_snowflake_packages(
+        self, temp_test_directory: str, streamlit_requirements_txt
+    ):
+        os.chdir(temp_test_directory)
+        os.rename(streamlit_requirements_txt, self.REQUIREMENTS_SNOWFLAKE)
         result = utils.get_snowflake_packages()
-        os.chdir("..")
 
         assert result == test_data.requirements
 
-    def test_get_snowflake_packages_delta(self, streamlit_requirements_txt):
+    def test_get_snowflake_packages_delta(
+        self, temp_test_directory: str, streamlit_requirements_txt
+    ):
         anaconda_package = test_data.requirements[-1]
-        os.chdir(".tests")
+        os.chdir(temp_test_directory)
+        os.rename(streamlit_requirements_txt, self.REQUIREMENTS_SNOWFLAKE)
```

Comment:
Why do we firstly create file with random name to change it later to a specific name?

Reply:
If you need a file with a specific name and if you create it in temp directory (what takes care of cleaning), then IMO it's better to create a file with the target name right up front.